plugins/ingest-attachment/src/test/resources/rest-api-spec/test/ingest_attachment/10_basic.yaml

```diff
@@ -8,4 +8,19 @@
         nodes.info: {}
 
     - match:  { nodes.$master.plugins.0.name: ingest-attachment  }
-    - match:  { nodes.$master.ingest.processors.11.type: attachment }
+    - match:  { nodes.$master.ingest.processors.0.type: append }
```

Comment:
what do these processors have to do with attachment? Why can't these checks stay in the general rest tests?